dql/history.py

```diff
@@ -0,0 +1,49 @@
+import os
+from pathlib import Path
+
+
+class HistoryManager(object):
+    _initial_history_length = 0
+
+    def _create_dir_if_not_exists(self, path: str) -> None:
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+    def _create_file_if_nost_exists(self, path: str) -> None:
+        if not os.path.exists(path):
+            with open(path, "w"):
+                pass
+
+    def _get_history_dir_and_file(self):
+        home = str(Path.home())
```

Comment:
You don't have to `str()` this, or even store it in a variable since you're just path-joining it in the line below